project.clj

```diff
@@ -78,7 +78,8 @@
                  [ring/ring-json "0.4.0"]                             ; Ring middleware for reading/writing JSON automatically
                  [stencil "0.5.0"]                                    ; Mustache templates for Clojure
                  [toucan "1.0.2"                                      ; Model layer, hydration, and DB utilities
-                  :exclusions [honeysql]]]
+                  :exclusions [honeysql]]
+                 [org.clojars.pntblnk/clj-ldap "0.0.12"]]             ; LDAP library
```

Comment:
I redid this in the merge conflict resolution branch metabase/metabase:ldap-auth